app/controllers/decks.rb

```diff
@@ -12,11 +13,19 @@
 	deck.cards.each do |card|
 		Guess.create(round: round, card: card)
 	end
-	redirect '/decks/deck.id/cards'
+
+	redirect '/decks/:deck_id/cards'
```

Comment:
i changed this back to :deck_id. but i think we should be referencing the instances id (deck.id) but i was getting an error (regenerating shotgun, will post error in a moment)